config/kustomize/examples/transformer/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 configurations:
-- https://raw.githubusercontent.com/numaproj/numaplane/main/config/kustomize/numaplane-transformer-config.yaml
```

Comment:
I chose to use the URL in case a user chooses to only download the examples directory but I'm ok if you want to change this.

Reply:
Actually let me switch it back.